src/components/Card/Card.js

```diff
@@ -1,42 +1,75 @@
 import { useLocation } from 'react-router-dom';
+import { convertDate } from '../../utils/utilFunctions';
 import './Card.css';
 
 const Card = (props) => {
-  const { keyword, image, date, title, text, source, isLoggedIn } = props;
+  const { card, isLoggedIn, onSaveCardClick, isLoggedInSavedNews } = props;
   const location = useLocation();
 
-  console.log(isLoggedIn);
+  const handleSaveCardClick = () => {
+    onSaveCardClick(card);
+  };
 
   return (
     <>
```

Comment:
` <>` empty tags are not needed if you have a common tag that wraps all the markup of the component